### PR TITLE
Add SECURITY.md

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
Adds SECURITY.md which outlines vulnerability policy.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Added security policy
```

### CODE REVIEW

**Comment by awels** [comment]
ci test please

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| master  | Yes                |
+| >= 1.9.0| Yes                |
```
Comment (aglitke) [inline SECURITY.md]: So after 1.9.1 is released we still support 1.9.0?

Comment (awels) [reply SECURITY.md]: Well would think so, any security updates for 1.9.x would go into 1.9.y where y is a new number.

**Comment by j-griffith** [comment]
@aglitke you good with this now?

**Comment by awels** [comment]
I am going to close this, unless someone objects, can reopen then.

**Pull request closed without merging**